packages/openapi-v3/src/__tests__/integration/controller-spec.integration.ts

```diff
@@ -332,6 +333,66 @@ describe('controller spec', () => {
     expect(globalSchemas).to.be.empty();
   });
 
+  it('allows a class to provide definitions of referenced models through #/components/schemas', () => {
+    @api({
+      paths: {
```

Comment:
What I meant is to only have schemas at `@api` level for the controller class, not individual operations.